oxidation/libparsec/crates/protocol/schema/authenticated_cmds/vlob_create.json

```diff
@@ -116,6 +108,38 @@
                     "type": "List<Bytes>"
                 }
             ]
+        },
+        {
+            "status": "sequester_rejected",
+            "other_fields": [{
+                    "name": "service_id",
+                    "type": "SequesterServiceID"
+                },
+                {
+                    "name": "service_label",
+                    "type": "String"
+                },
+                {
+                    "name": "service_error",
+                    "type": "String"
+                }
+            ]
+        },
+        {
+            "status": "sequester_webhook_failed",
```

Comment:
Should we make this a more generic error ?
from my point of view the sequester can provided two kind of errors:
1) temporary failure in the sequester (for instance the webhook couldn't be reached), this error means the client should retry the sync later on.
2) hard failure in the sequester: a sequester service explicitly said this manifest is not to be synced.

For 1), a very simple implementation for this is for the server to close the client socket if it occurs (this way not change in the client and no need for a specific status \o/).

For 2) things get more complicated: the client should stop trying to sync the corresponding file (I believe this PR already implement this) AND the other sequester services that have been notified with the data should receive an erratum (given they have stored a given vlob_atom that is in fact not present in the server database !)
The simple solution to this is to consider the sequester services trust the server and replace the vlob_atom if a new one with same vlobid/version is sent to them (so there is nothing to do for webhook service given we store nothing, but for the storage service we must change the sql query a bit and add a test)

In any case we should try to avoid avoid an error specific to a kind of sequester service, this would simplify adding new type of sequester in the future.